### ISSUE TITLE
Always returned NULL

### ISSUE THREAD

**Issue by OFarukCaki** [fransyozef/php-bittrex#1]
Hello,

I'm trying but I couldn't find what is my problem. I try to run any of those methods but always I'm getting NULL as returned. What would be problem?

Thanks in advance

**Comment by fransyozef** [comment]
have you activiated the api keys in Bittrex and added them in config.inc.php?

**Comment by OFarukCaki** [comment]
Yes I did, I already using same API keys with Node.js and working fine, But I couldn't make it running via PHP. By the way what is the PHP version and am I need to install any other extensions like curl etc. ?

**Comment by fransyozef** [comment]
On my localhost I'm using

PHP Version 5.3.1

Curl information:

cURL support 	enabled
cURL Information 	7.43.0
Age 	3
Features
AsynchDNS 	Yes
Debug 	No
GSS-Negotiate 	No
IDN 	No
IPv6 	Yes
Largefile 	Yes
NTLM 	Yes
SPNEGO 	Yes
SSL 	Yes
krb4 	No
libz 	Yes
Protocols 	dict, file, ftp, ftps, gopher, http, https, imap, imaps, ldap, ldaps, pop3, pop3s, rtsp, smb, smbs, smtp, smtps, telnet, tftp
Host 	i386-apple-darwin15.0
SSL Version 	SecureTransport
ZLib Version 	1.2.5

**Comment by fransyozef** [comment]
So yes .. you need curl for php

**Comment by fransyozef** [comment]
I think I found the problem. The NULL value is when you use the functions from CLI. I made a patch and pushed it into https://github.com/fransyozef/php-bittrex/commit/5dacce39e7ce1e8e4d2a8d83b12f001b0c18cd92
